_rules/meta-refresh-no-delay-no-exception-bisz58.md

```diff
@@ -0,0 +1,220 @@
+---
+id: bisz58
+name: '`meta` element has no refresh delay (no exception)'
+rule_type: atomic
+description: |
+  This rule checks that the `meta` element is not used for delayed redirecting or refreshing.
+accessibility_requirements:
+  wcag20:2.2.4: # Interruptions (AAA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:3.2.5: # Change on Request (AAA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G110: # Using an instant client-side redirect
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:H76: # Using meta refresh to create an instant client-side redirect
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'title-require'
+---
+
+## Applicability
+
+This rule applies to the first `meta` element in a document for which all the following are true:
+
+- the element has an `http-equiv` [attribute value][] of `"refresh"`; and
+- the element has a [valid][valid content] `content` [attribute value][].
+
+## Expectation
+
+For each test target, running the [shared declarative refresh steps][], given the target's document, the value of the target's `content` attribute, and the target results in _time_ being either 0 or greater than 72000 (20 hours).
```

Comment:
I don't think this rule should have the 20 hours exception, since this is the "no exception" rule (and the 20 hours exception comes from SC 2.2.1 which is not checked in this rule).

Reply:
🙈 copy/pasting for the win (or loss)!